Back-End-Node/vercel.json

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
why is this file deleted? I think we need it for deployment

Reply:
this is a result of not commiting constantly, and this branch was written before versel was created